docs/cuj/exercises/exercise1/Exercise1.md

```diff
@@ -0,0 +1,5 @@
+# Definition
+Assume you are a new kubernetes developer and you need write a CI/CD pipeline for your application.
+You stumbled upon this project and want to evavluate if this a possible solution for defining a CI/CD pipeline.
+
+Read the [Pipeline CRD Concepts](./../../../Concepts.md) to understand this project.
```

Comment:
Should we point them at the [using](https://github.com/knative/build-pipeline/blob/e8343b988799b33fca2862379790c4b14d0b30a8/docs/using.md) docs as well now that we have those?

And maybe the [README](https://github.com/knative/build-pipeline/tree/e8343b988799b33fca2862379790c4b14d0b30a8#-pipeline-crd) as well?

Reply:
I am against The using docs now right now. The goal of this User study is to get feedback on DSL.
We could conduct last user study on testing the using docs and tutorial.
We should probably do it with someone who already understands pipeline CRD so that we still make it under 45 mins time limit.
How does that sound?

Reply:
kk that's fine!